financial-connections/src/main/java/com/stripe/android/financialconnections/presentation/ComposeExtensions.kt

```diff
@@ -1,3 +1,5 @@
+@file:Suppress("ComposeCollectAsStateUsageIssue")
```

Comment:
Only one I didn't replace in production code. Financial connections has a custom `collectAsState` here that has additional logic.